rto_node/launch/robotino_driver.launch.py

```diff
@@ -165,6 +169,16 @@ def generate_launch_description():
         'launch_teleopnode',
         default_value='true',
         description= 'Weather to start teleop node not based on launch environment')
+    
+    declare_bumper_timeout_argument = DeclareLaunchArgument(
```

Comment:
These defaults are not the same as in the other launch file (and in the src file declaration). They should be consistent.

I like the bumper timeout of 2 sec (in the other cases it is 5 sec).